src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderCursor.cs

```diff
@@ -682,9 +695,9 @@ private bool IsSep(char ch)
                     }
                 }
 
-                private void ThreadProc()
+                private void ReadLines()
                 {
```

Comment:
suggestion: ThreadProc indicates that it is the main procedure of the of the thread. It might be better to retain that meaning and call it ReadLinesThreadProc